demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -190,15 +190,23 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
     } else {
       (document.getElementById('inputMeeting') as HTMLInputElement).focus();
     }
-    this.defaultBrowserBehaviour = new DefaultBrowserBehavior();
   }
 
   initEventListeners(): void {
+    if (!this.defaultBrowserBehaviour.hasChromiumWebRTC()) {
+      (document.getElementById('simulcast') as HTMLInputElement).disabled = true;
+      (document.getElementById('planB') as HTMLInputElement).disabled = true;
+    }
+    
     document.getElementById('form-authenticate').addEventListener('submit', e => {
       e.preventDefault();
       this.meeting = (document.getElementById('inputMeeting') as HTMLInputElement).value;
       this.name = (document.getElementById('inputName') as HTMLInputElement).value;
       this.region = (document.getElementById('inputRegion') as HTMLInputElement).value;
+      this.enableSimulcast = (document.getElementById('simulcast') as HTMLInputElement).checked;
+      this.enableWebAudio = (document.getElementById('webaudio') as HTMLInputElement).checked;
+      // js sdk default is true
+      this.enableUnifiedPlanForChromiumBasedBrowsers = !(document.getElementById('planB') as HTMLInputElement).checked;;
```

Comment:
Double `;;`.

If the default is true, you should probably default the input box as `checked`, no?

Reply:
"Disable Unified Plan for Chrome" is default unchecked equivalent to  "MeetingSessionConfiguration" has "enableUnifiedPlanForChromiumBasedBrowsers" to be true.
The selection is "Disable Unified Plan for Chrome" since developer could still disable it by setting the field in MeetingSessionConfiguration to false.